### PR TITLE
Baseline v5: variance measurement with curvature proxy code

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
Need a baseline rerun on the latest code (with curvature proxy) to calibrate noise floor. Previous reruns were on old code.

## Instructions
NO changes. Run unmodified: `python train.py --agent kohaku --wandb_name "kohaku/baseline-v5" --wandb_group baseline-v5-variance`
## Baseline: val/loss=2.1997
---
## Results

**W&B run:** `ex1vpoi0`
**Epochs completed:** 64 (30-min timeout)
**Peak memory:** N/A (system metrics not captured — run timed out without wandb.finish())

### Metrics at best checkpoint (epoch 64)

| Split | val/loss | surf Ux | surf Uy | surf p | vol Ux | vol Uy | vol p |
|---|---|---|---|---|---|---|---|
| val_in_dist | 1.5840 | 0.2920 | 0.1803 | 21.80 | 1.2823 | 0.4702 | 26.90 |
| val_ood_cond | 1.9186 | 0.2686 | 0.1893 | 20.59 | 1.0770 | 0.4121 | 20.19 |
| val_tandem_transfer | 3.2673 | 0.6256 | 0.3343 | 41.61 | 2.1315 | 0.9808 | 43.42 |
| **3-split mean** | **2.2566** | — | — | — | — | — | — |

### Noise floor calibration

This baseline-v5 run: **2.2566**
Previous baseline (v4, which introduced curvature proxy): **2.1997**
Delta: +0.057 (worse than v4)

This is ~1.8× the typical seed std (±0.032). Combined with the v4 measurement, we now have two data points on the curvature-proxy code baseline:

| Run | val/loss_3split |
|---|---|
| curvature baseline (v4) | 2.1997 |
| baseline-v5 (this run) | 2.2566 |
| mean | ~2.228 |
| range | 0.057 |

The v4 result now looks like a lucky run. The updated noise floor is roughly 2.23 ± 0.03, with occasional lucky runs reaching ~2.20. Any experiment result claiming improvement should beat ~2.21 to be convincing.

### Suggested follow-ups

- One more seed run would help pin down the noise floor (current spread 0.057 across 2 runs is too wide for confident comparisons).
- Results from PR #930 (checkpoint averaging, val/loss=2.2755) now look worse: it's above both baseline samples. Checkpoint averaging appears to actively hurt.

### CODE REVIEW

**Comment by tcapelle** [comment]
## Review — Closing
Does not beat baseline (2.1997). Closing.
